index.html

```diff
@@ -63,5 +63,143 @@ <h2 class="let">LET'S CONNECT</h2>
+          <ul class="card__content--item1">
+            <li class="card__item--heading"><img src="" alt=""> Canopy</li>
+            <li> <img src="images/Counter.png" alt=""> Back End Dev</li>
+            <li> <img src="images/Counter.png" alt=""> 2015</li>
+          </ul>
+          <p class="card__content--desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="card__content--item2">
+            <li class="tech__btn">html</li>
+            <li class="tech__btn">css</li>
+            <li class="tech__btn">Javascript</li>
+          </ul>
+          <button class="btn">See Project</button>
+        </div>
+      <section class="ab">
+        <div class="about">
+  <div class="about__content">
+    <h2 class="aheading">About Myself</h2>
+    <p class="ades">
+      Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+    </p>
+    <p class="aget">LET'S CONNECT</p>
+    <ul class="socialmediaicons" >
+      <li>
+        <a class="social" href="https://twitter.com/hotumanzade">
+          <img src="Images\twitter.png" alt="twitter">
+        </a>
+      </li>
+      <li>
+        <a class="social" href="https://www.linkedin.com/in/burak-otman-88646443/">
+          <img src="Images\Linkedin icon.png" alt="link"> </a>
+        </a>
+      </li>
+      <li>
+        <a class="social" href="#">
+          <img src="Images\m.png" alt="Angelist"> </a>
+        </a>
+      </li>
+      <li>
+        <a class="social" href="https://github.com/otmanTR">
+          <img src="Images\kedi.png" alt="github"> </a>
+        </a>
+      </li>
+      <li>
+        <a class="social" href="#">
+          <img src="Images\vin.png" alt="medium"> </a>
+        </a>
+      </li>
+   </ul>
+   <button class="btn">Get my resume</button>
+  </div>
+  <div class="About__skills">
+    <ul class="lan"><li class="lan1">Languages <img class="disable3" src="/Images/Vektor.png"></li>
+      <ul class="imm">
+        <ul class="im">
+          <li class="imagess"><img  src="Images/Ellipse 1.png" alt=""></li>
+          <li class="wordss">Javascript</li></ul>
+        <ul class="im">
+          <li class="imagess"><img  src="Images/Ellipse 1 (1).png" alt=""></li>
+          <li class="wordss">HTML</li></ul>
+        <ul class="im">
+          <li class="imagess"><img  src="Images/Ellipse 1 (2).png" alt=""></li>
+          <li class="wordss">CSS</li></ul></ul>
+        <li class="lan1">Frameworks <img src="/Images/Disabled.png" class="disable1" alt="#"></li>
+        <img class="Seperatorimg1" src="/Images/Separator Bottom.png" alt="#">
+        <li class="lan1">Skills <img src="/Images/Disabled.png" class="disable2" alt="#"></li>
+        <img class="Seperatorimg2" src="/Images/Separator Bottom.png" alt="#">
```

Comment:
- [ ] Please check through your `img` elements and kindly remove the` /`, as it is the images are not yet displayed. 👍